packages/@wmr-plugins_nomodule/index.js

```diff
@@ -1,10 +1,17 @@
-import babel from '@babel/standalone';
+import { transform } from '@swc/core';
+import Visitor from "@swc/core/Visitor";
 
 /** @param {import('../../types').Options} options */
 export default function (options) {
 	if (options.mode && options.mode !== 'build') return;
 	options.plugins.push(nomodulePlugin({}));
 }
+class LegacyRewriter extends Visitor {
+  visitImportDeclaration(e: CallExpression): Expression {
```

Comment:
Not fully compatible yet with the previous version of this afaik, is it the intention that we rewrite every import to `.legacy.js`?

Reply:
yep.